CMakeLists.txt

```diff
@@ -394,6 +395,30 @@ IF (BUILD_STATIC_LIBRARY)
 
 ENDIF ()
 
+IF (BUILD_UTILITY)
+
+    SET (UTILITY_NAME "${PROJECT_PACKAGE_NAME}")
+    SET (UTILITY_TARGET "${UTILITY_NAME}.exe")
+
+    FILE (GLOB_RECURSE UTILITY_HEADERS "${PROJECT_SOURCE_DIR}/include/${PROJECT_PATH}/*.hpp")
+    FILE (GLOB_RECURSE UTILITY_SRCS "${PROJECT_SOURCE_DIR}/share/utils/*.cxx")
```

Comment:
Do we ever put stuff in the /share dir? I've never seen it used

Reply:
This is not in /share, but in the main project dir `/share/utils/*.cxx`
I can move it to another folder, I was just mirroring what we had in OSTk core.
Maybe in project_srouce_dir/scripts/

Reply:
Yeah that's what I meant, I've never seen us use the main project dir /share before, should we depracate that folder? I do indeed prefer putting it in either in a /scripts dir or even in the /test dir

Reply:
Yeah /scripts might be good.